src/docs/010_manual/20_install.adoc

```diff
@@ -89,32 +98,37 @@ To get started, you can run `tasks` as a first command:
 ----
 ./dtcw.ps1 tasks --group=doctoolchain
 ----
+
+NOTE: if you're making use of cmd.exe, please replace `.ps1` with `.bat`. Do this for subsequent commands.
+
 --
 
-This command will show you all tasks available through docToolchain, but first it will make sure that docToolchain is installed:
+This 'tasks' command display you all of the tasks available on docToolchain. It also verify that docToolchain is installed by:
+
+* Checking for a configuration file. It will creates a default one if none is available. It prefers a local installation. 
+
+* Making use of docker if it is available and there is no local installation. Please keep in mind that if this is your first time running docToolchain, the images download could take a while.
 
-* it checks for a configuration file and creates a default one if none is available.
-* it prefers a local installation
-* if docker is available and no local installation, it will use docker. The download of the images might take a while when run for the first time
 * if docker is not available, it will download and install docToolchain to your home folder
-** it will then run a gradle build through the gradle wrapper. This will download and install gradle when run for the first time.
-** the gradle build will also fetch some dependencies. This will also take some time when run for the first time.
+** it will then run a gradle build through the gradle wrapper. If this is your first time, It will download and install gradle.
+** the gradle build will also fetch some dependencies. This will also take a while if ran for the first time.
 
-As you can see, the installation is an incremental process and will take some time because of additional downloads.
-After the first few commands, you will have all the needed dependencies installed and things will be faster.
+You'll notice the installation is an incremental process and will take a while because of all the additional dependencies needed.
+After the first few commands, you'll have all the needed dependencies installed and things will run more quickly.
```

Comment:
After the first few commands, you'll have all the required dependencies installed and things will run faster.


@Bennykillua Just grammar again. Really like the way you changed the sentence structure here. :)